app/assets/javascripts/components/export_case/export_case_modal_instance_controller.js

```diff
@@ -47,5 +47,25 @@ angular.module('QuepidApp')
       ctrl.cancel = function () {
         $uibModalInstance.dismiss('cancel');
       };
+      
+      
+      ctrl.isProcessingFile = function () {
+        if (ctrl.options.snapshot){
+          var desiredSnapshot = null;
+          angular.forEach(ctrl.snapshots, function(snapshot) {
+            if (snapshot.id === ctrl.selection) {
+              desiredSnapshot = snapshot;
+              return; // exit the loop early
```

Comment:
If you return when you set desiredSnapshot, you won't ever get to line 61 where you return hasSnapshotFile. Either this should be a break, or lines 61-66 can be deleted and you return desiredSnapshot.hasSnapshotFile (or just snapshot.hasSnapshotFile, as the variable is redundant.

Reply:
Fixed!   thank you!